cci_nepal/pipeline/dummy_data/model_complete.py

```diff
@@ -0,0 +1,367 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.13.6
+#   kernelspec:
+#     display_name: cci_nepal
+#     language: python
+#     name: cci_nepal
+# ---
+
+# +
+import pandas as pd
+import numpy as np
+
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import MinMaxScaler
+from sklearn.metrics import accuracy_score
+from sklearn.model_selection import cross_val_score
+from sklearn.model_selection import RepeatedKFold
+from sklearn.feature_selection import SequentialFeatureSelector
+
+
+from matplotlib import pyplot as plt
+import time
+from operator import itemgetter
+
+import logging
+import cci_nepal
+from cci_nepal.getters.dummy_data import get_dummy_data as gdd
+from cci_nepal.utils.dummy_data import dummy_data_utils as ddu
+
+# %load_ext nb_black
+
+project_dir = cci_nepal.PROJECT_DIR
+logging.info(project_dir)
+
+
+df = gdd.read_csv_file(f"{project_dir}/inputs/data/Dummy_Data_Full_Survey_Features.csv")
+df.head()
```

Comment:
As this is a pipeline script its more functional rather than explorative so you can remove `.head()` cases. I think its supposed to hold a series of functions than can just be run to produce an output